root/documentation/ld.conf

```diff
@@ -1,7 +1,7 @@
 <endpoints>
   <ld_id_get>
     description=Computes and returns LD values between the given variant and all other variants in a window centered around the given variant. The window size is set to 500 kb.    
-    endpoint=ld/:species/:id/:population_name/:d_prime/:r2/:window_size
+    endpoint=ld/:species/:id/:population_name/:d_prime/:r2/:window_size/:attribs
```

Comment:
I see. Only the required parameters are listed here? then I also need to remove :d_prime/:r2/:window_size. Is that correct?